packaging/config/agent.conf

```diff
@@ -0,0 +1,5 @@
+--package-directory=/var/lib/stackable/packages
+--config-directory=/var/lib/stackable/config
+--data-directory=/var/lib/stackable/agentdata
```

Comment:
Why the extra "data" at the end? Wouldn't `/var/lib/stackable/agent` make more sense?